src/Model.js

```diff
@@ -2,7 +2,14 @@ import _ from 'lodash';
 
 export default class Model {
   constructor(attributes) {
-    this.attributes = attributes;
+    this.attributes = {};
```

Comment:
thanks, I have update. Regarding references, the alternative solution is cloneDeep but it can cause the performance. I think it is matter of discuss.